src/components/preambles.py

```diff
@@ -446,19 +446,19 @@ def preamble_device_address_set(transport, idx, trace):
     Scramble the company_id part of the Bluetooth address.
 """
 def address_scramble_OUI(address):
-    return (address & 0x00ff00ffffff) | ((address << 16) & 0xff0000000000) | ((address >> 16) & 0x0000ff000000);
+    return (address & 0x00ff00ffffff) | (~address & 0xff0000000000) | (~address & 0x0000ff000000);
```

Comment:
As far as I can tell, this could be written more concisely as `address ^ 0xff00ff000000`; similarly below.

Reply:
Agree - will update